test/test_stochastic_rounding.py

```diff
@@ -0,0 +1,28 @@
+import math
```

Comment:
If you add a new test file, you should add it to https://github.com/pytorch/pytorch/blob/master/test/run_test.py

Reply:
Thank you!